src/Application/Console/Definitions/AnalyseDefinition.php

```diff
@@ -63,6 +63,13 @@ public static function get(): array
                 InputOption::VALUE_NONE,
                 'Disable Security issues check to not throw error if vulnerability is found'
             ),
+            new InputOption(
+              'format',
+              null,
+              InputOption::VALUE_REQUIRED,
+                'Format to output the result in',
```

Comment:
Can we have a list of value possible ? (console, json)

Reply:
Is there a build in way to do that? or just add in the the comment for the option?